packages/blacklens/manifest.yml

```
@@ -0,0 +1,100 @@
+format_version: 3.3.0
+name: blacklens
+title: "blacklens.io"
+version: 0.0.1
+source:
+  license: "Elastic-2.0"
+description: "Collect logs from blacklens.io with Elastic Agent"
+type: integration
+categories:
+  - security
+conditions:
+  kibana:
+    version: "^8.15.2"
+  elastic:
+    subscription: "basic"
+screenshots:
+  - src: /img/blacklens-dashboard.png
```

```suggestion
  - src: /img/blacklens-dashboard-alerts.png
```

Rename the file accordingly.